model/models/apertus/model.go

```diff
@@ -0,0 +1,212 @@
+	// - alphaN: Scaling factor for negative inputs (default: 0.5)
+	// - alphaP: Scaling factor for positive inputs (default: 1.0)
+	// - beta: Shape parameter for the exponential curve (default: 1.0)
+	// - epsIELU: Small epsilon value for numerical stability (default: 1e-12)
+	alphaN, alphaP, beta, epsIELU float32
+}
+
+type Model struct {
+	model.Base
+	model.TextProcessor
+
+	TokenEmbedding *nn.Embedding `gguf:"token_embd"`
+	Layers         []Layer       `gguf:"blk"`
+	OutputNorm     *nn.RMSNorm   `gguf:"output_norm"`
+	Output         *nn.Linear    `gguf:"output,alt:token_embd"`
+
+	Options
+}
+
+func New(c fs.Config) (model.Model, error) {
+	if c.Uint("expert_count") > 0 {
+		// Apertus architecture doesn't support mixture of experts.
+		// If a model file claims to be Apertus but has expert_count > 0,
+		// it's likely misidentified or corrupted.
+		return nil, model.ErrUnsupportedModel
+	}
+
+	var processor model.TextProcessor
+	vocabulary := model.Vocabulary{
+		Values: c.Strings("tokenizer.ggml.tokens"),
+		Scores: c.Floats("tokenizer.ggml.scores"),
+		Types:  c.Ints("tokenizer.ggml.token_type"),
+		Merges: c.Strings("tokenizer.ggml.merges"),
+		AddBOS: c.Bool("tokenizer.ggml.add_bos_token", true),
+		BOS:    []int32{int32(c.Uint("tokenizer.ggml.bos_token_id"))},
+		AddEOS: c.Bool("tokenizer.ggml.add_eos_token", false),
+		EOS: append(
+			[]int32{int32(c.Uint("tokenizer.ggml.eos_token_id"))},
+			c.Ints("tokenizer.ggml.eos_token_ids")...,
+		),
+	}
+	// Apertus models use GPT2 byte-level BPE tokenizer with tekken pretokenizer
+	switch c.String("tokenizer.ggml.model") {
+	case "gpt2":
+		var pretokenizers []string
+		switch c.String("tokenizer.ggml.pre") {
+		case "tekken":
+			pretokenizers = []string{
+				"[^\\r\\n\\p{L}\\p{N}]?[\\p{Lu}\\p{Lt}\\p{Lm}\\p{Lo}\\p{M}]*[\\p{Ll}\\p{Lm}\\p{Lo}\\p{M}]+|[^\\r\\n\\p{L}\\p{N}]?[\\p{Lu}\\p{Lt}\\p{Lm}\\p{Lo}\\p{M}]+[\\p{Ll}\\p{Lm}\\p{Lo}\\p{M}]*|\\p{N}| ?[^\\s\\p{L}\\p{N}]+[\\r\\n/]*|\\s*[\\r\\n]+|\\s+(?!\\S)|\\s+",
+			}
+		default:
+			return nil, model.ErrUnsupportedTokenizer
+		}
+		processor = model.NewBytePairEncoding(&vocabulary, pretokenizers...)
+	default:
+		return nil, model.ErrUnsupportedTokenizer
+	}
+
+	m := Model{
+		TextProcessor: processor,
+		Layers:        make([]Layer, c.Uint("block_count")),
+		Options: Options{
+			hiddenSize: int(c.Uint("embedding_length")),
+			numHeads:   int(c.Uint("attention.head_count")),
+			numKVHeads: int(c.Uint("attention.head_count_kv")),
+			headDim:    int(c.Uint("attention.key_length")),
+			ropeDim:    int(c.Uint("rope.dimension_count")),
+			eps:        c.Float("attention.layer_norm_rms_epsilon"),
+			ropeBase:   c.Float("rope.freq_base", 1e5),
+			ropeScale:  c.Float("rope.scaling.factor", 1),
+			// Load xIELU parameters with defaults
+			alphaN:  c.Float("xielu.alpha_n", 0.5),
+			alphaP:  c.Float("xielu.alpha_p", 1.0),
+			beta:    c.Float("xielu.beta", 1.0),
+			epsIELU: c.Float("xielu.eps", 1e-12),
```

Comment:
based on [existing ggml models][1], these are ggml arrays so it should use `c.Floats(...)`

also since these keys don't start with `general` or `tokenizer`, `c.Floats` will prepend the architecture so it ends up looking for `apertus.xielu.alpha_n`, for example, which isn't how it's stored in the file

[1]: https://huggingface.co/unsloth/Apertus-8B-Instruct-2509-GGUF/tree/main?show_file_info=Apertus-8B-Instruct-2509-BF16.gguf